thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -20,7 +20,9 @@ export default Component.extend({
   classNames: ['rootcause-anomaly'],
   entities: null, // {}
 
-  anomalyUrn: null, // ""
+  aggregates: null, // {}
```

Comment:
I'm confused as to what the comment beside this is. Is that a default value? If so, jsdocs has a good format for that
```
/**
 * Your description
 * @type {type}
 * @default value, ex: 'Han shot first' or {}
 */
yourProperty: null,
```

Reply:
That's a default value for a variable that's expected its value to be passed by the parent component. I'll update the jsdocs for this.